nautobot/docs/development/testing.md

```diff
@@ -61,15 +61,38 @@ Nautobot's custom [test runner](https://docs.djangoproject.com/en/3.2/topics/tes
 
-`NautobotPerformanceTestRunner` which inherits from `DiscoverSlowestTestsRunner` will only be available when you have `django-slowtests` installed and set `TEST_GENERATE_PERFORMANCE_REPORT` in `settings.py` to True. When running performance tests, `NautobotPerformanceTestRunner` will replace the default test runner `NautobotTestRunner` and will output a performance evaluation at the end of each run.
+!!! info
+    `invoke performance-test` is enabled when `django-slowtests` is installed and when called, it will run and evaluate the performance of specific unittests that are tagged with `performance` i.e. `@tag("performance")`. `invoke unittest --performance-report` and `invoke integration-test --performance-report` will also be enabled and when called, they will generate a performance report for all the tests ran in the terminal.
+    If performance baselines for tests are not available:
 
-An example output of the performance evaluation would be:
+```no-highlight
+175 abnormally slower tests:
+Performance baseline for test_account (nautobot.circuits.tests.test_filters.ProviderTestCase) is not available. Test took 0.0758s to run
+Performance baseline for test_asn (nautobot.circuits.tests.test_filters.ProviderTestCase) is not available. Test took 0.0427s to run
+Performance baseline for test_bulk_create_objects (nautobot.circuits.tests.test_api.CircuitTerminationTest) is not available. Test took 0.2900s to run
+Performance baseline for test_bulk_create_objects (nautobot.circuits.tests.test_api.CircuitTest) is not available. Test took 0.2292s to run
+Performance baseline for test_bulk_create_objects (nautobot.circuits.tests.test_api.CircuitTypeTest) is not available. Test took 0.1596s to run
+Performance baseline for test_bulk_create_objects (nautobot.circuits.tests.test_api.ProviderNetworkTest) is not available. Test took 0.1897s to run
+Performance baseline for test_bulk_create_objects (nautobot.circuits.tests.test_api.ProviderTest) is not available. Test took 0.2092s to run
+Performance baseline for test_bulk_delete_objects (nautobot.circuits.tests.test_api.CircuitTerminationTest) is not available. Test took 0.1168s to run
+Performance baseline for test_bulk_delete_objects (nautobot.circuits.tests.test_api.CircuitTest) is not available. Test took 0.2762s to run
+Performance baseline for test_bulk_delete_objects (nautobot.circuits.tests.test_api.CircuitTypeTest) is not available. Test took 0.0663s to run
+Performance baseline for test_bulk_delete_objects (nautobot.circuits.tests.test_api.ProviderNetworkTest) is not available. Test took 0.0875s to run
+...
+```
+
+!!! info
+    If performance baselines for tests are available and the time it take to run tests are siginificantly slower than baselines:
```

Comment:
Good step forward, and I see that you moved the all of the `TEST_GENERATE_PERFORMANCE_REPORT` docs here, but what seems to be missing are any references to passing `--testrunner` or setting the `TEST_RUNNER` in your config?

Perhaps that would be best in the `development/testing.md` file somewhere?